pkg/aws/rds/rds.go

```diff
@@ -73,15 +74,25 @@ func (r RDS) CreateDBCluster(ctx context.Context, storage int64, instanceClass,
 	return r.CreateDBClusterWithContext(ctx, dbi)
 }
 
+func (r RDS) CreateDBInstanceInClusterForTest(ctx context.Context, restoredClusterID, instanceID, instanceClass, dbEngine, dbSubnetGroup string, publicAccess bool) (*rds.CreateDBInstanceOutput, error) {
```

Comment:
Can you explain the difference between these three methods? I see CreateDBInstance, InCluster and InClusterForTest.

Reply:
CreateDBInstance -> creates a DB instance, this is used for RDS postgres apps test
InCluster -> creates a DB instance using cluster ID, this is required when we spin up instance during restore, so in argument it also accepts a clusterID

InClusterForTest -> same as above(2) but this has `publiclyAccessible` field set to false.